lib/game.rb

```diff
@@ -170,9 +141,11 @@ def restart_game
   end
 
   def play_game
+    computer_placement
```

Comment:
Computer and Player placement in play_game reads better. Good integration and movement! 👍